nncf/experimental/torch/sparsity/movement/structured_mask_handler.py

```diff
@@ -320,6 +337,9 @@ def populate_dependent_structured_mask_to_operand(self):
         """
         Asks all contexts in `self._structured_mask_ctx_groups` to update the actual binary masks in operand.
         """
+        for group in self._structured_mask_ctx_groups:
+            for ctx in group.structured_mask_contexts:
+                ctx.initialize_binary_mask()
```

Comment:
@vuiseng9 @yujiepan-work At this point unstructured binary mask was overridden anyway. I just initialize it by ones first to apply inflated structured masks (per row or/and per column) by multiply (or logic AND).